### ISSUE TITLE
Tiny feature request: Breakpoint on domain should always add /* in breakpoint URL

### ISSUE THREAD

**Issue by carstenhag** [ProxymanApp/Proxyman#1334]
### Proxyman version? (Ex. Proxyman 1.4.3)
3.8.0

### macOS Version? (Ex. mac 10.14)
12.4

### Steps to reproduce 
<img width="525" alt="Bildschirmfoto 2022-08-17 um 10 51 17" src="https://user-images.githubusercontent.com/3776997/185077227-0426b06e-d0c8-46e6-8f2e-09b1c34f6cc1.png">
<img width="804" alt="Bildschirmfoto 2022-08-17 um 10 51 25" src="https://user-images.githubusercontent.com/3776997/185077293-0c9c1ac1-ea38-47e7-93e6-7f347d27ea4b.png">

### Expected behavior
When adding a breakpoint to a domain/subdomain, it is expected to intercept all calls with that domain/subdomain. But Proxyman is currently only adding "duckduckgo.com" to the matching URL. With that URL, you will only get exact matches. 

It should automatically add "duckduckgo.com/*" so that every request is intercepted.

**Comment by NghiaTranUIT** [comment]
I don't think that it's a good point.

`/*` is equivalent to "Include all subpaths of this URLs". I set it off by default because if it's ON, there are a lot of incoming breakpoints. Thus, the user has to select the one they need to do a breakpoint.

From my experience, most of the time, we would like to set a breakpoint for a single request. Thus, OFF is default is better 👍

**Comment by carstenhag** [comment]
For me as an app dev I never have any requests that I want to intercept at /. It's always /api/v2/some-call for example.  
I guess only for web devs it makes sense to specifically only target /.

Maybe it could be an option? :D.

**Comment by NghiaTranUIT** [comment]
yes, it's from the Web dev. When you reload a website at mydomain.com, there are a bunch of requests, such as mydomain.com. If you create a rule with `/*`, there are a bunch of breakpoints too. Not just the homepage `/`
